.github/workflows/main.yml

```diff
@@ -15,47 +15,75 @@ on:
 jobs:
```

Comment:
Can you please add the paths-ignore for the python tutorials and README here too?

Reply:
Ruff also lints the Jupyter notebooks (https://docs.astral.sh/ruff/configuration/#jupyter-notebook-discovery). Excluding the tutorials is not advised. I'm not sure if the README impacts the document rendering, so I left it out.

Reply:
Ah, ok.